acceptance-tests/src/test/resources/uk/gov/di/test/features/005_auth_app_2fa_journey.feature

```diff
@@ -2,42 +2,30 @@
 Feature: Authentication App Journeys
   New user creates an account and logs in using an auth app
 
-
-  Scenario: User successfully registers with auth app 2FA, logs out then signs in again
-    Given the registration services are running
-    And the auth app user has valid credentials
-    And the new user clears cookies
-    When the new user visits the stub relying party
-    And the existing user clicks "2fa-off"
```

Comment:
We've gotta cover registration without 2FA as it was a bug and it was added to the test suite for regression purposes.